fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStat.java

```diff
@@ -285,6 +285,17 @@ private double convertToDouble(Type type, String columnValue) throws AnalysisExc
 
     }
 
+    private double convertStringToDouble(String s) {
```

Comment:
I suggest that override `getLongValue` in StringLiteral, and move those logic to the overided method

Reply:
good idea